### PR TITLE
Return rejected promise when the document is not fully active, for operations returning promises.

### PR DESCRIPTION
**Pull request by padenot**
Essentially a conversion to bikeshed of @domenic's [comment](https://github.com/WebAudio/web-audio-api/issues/2176#issuecomment-640930168) and then used everywhere we create a promise.

***
<a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/pull/2210.html" title="Last updated on Jul 7, 2020, 9:02 AM UTC (c0ddda0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/2210/5d87cf8...c0ddda0.html" title="Last updated on Jul 7, 2020, 9:02 AM UTC (c0ddda0)">Diff</a>

### CODE REVIEW

**Comment by hoch** [comment]
I'd like to review this.

File: index.bs [hunk 1]
```diff
@@ -1105,9 +1105,11 @@ Methods</h4>
 			called, the following steps MUST be performed on the control
 			thread:</span>
 
-			1. Let <var>promise</var> be a new Promise.
+			1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=fully active=] then return [=a promise rejected with=] "{{InvalidStateError}}" {{DOMException}}.
```
[the remark this answers is not in the document]
Comment (padenot) [reply index.bs:1108]: The `[=something=]` notation refers to exported definitions that are globally exposed. I would have expected some sort of name-spacing per spec, but I don't know how to do it and haven't found anything.

When describing a method, `this` is the object on which the method has been called, I think it's clear enough.

When a document is not fully active, the promise is returned rejected, and it goes wherever the author of the code wants it to go.

**Comment by padenot** [comment]
> lgtm for the change.
> 
> But not sure what's up with bikeshed. In PR #2211, I've updated the expected results to match what I get locally, but Travis doesn't like it.

Probably a change in format for the errors. I've updated those based on `bikeshed` tip yesterday evening.

**Comment by hoch** [comment]
Thanks for the explanation. Still LGTM.

**Pull request merged**